Sources/CryptorECC/EllipticCurve.swift

```diff
@@ -133,3 +133,40 @@ public struct EllipticCurve: Equatable, CustomStringConvertible {
         return Data(bytes: hash)
     }
 }
+
+extension String {
+    
+    ///
+    /// Split a string to a specified length.
+    ///
+    ///    - Parameters:
+    ///        - length:                Length of each split string.
+    ///
+    ///    - Returns:                    `[String]` containing each string.
+    ///
+    func split(to length: Int) -> [String] {
```

Comment:
There's almost certainly a more efficient way of doing this using String.Index and String.insert, but that can wait for a future PR.